assets/common.sh

```diff
@@ -23,7 +42,7 @@ login() {
   local team=$4
   local insecure=$5
   local tried=$6
-  local target=$7
+  local target=main
```

Comment:
I'd like to understand why we're restricted to the `main` team with this change.

Reply:
b/c that variable is never passed in, that was a quick hack to make it work

Reply:
@troykinsella Can you review this when you get time, this change is required for concourse 5.2 or above.

Reply:
any chance on merging?

Reply:
@ChipWolf I think the correct course of action here is to actually pass through the target, rather than putting a band-aid on the local target variable. This has been fixed on master, so please revert this change.